app/src/main/res/layout/view_avatar_name_email.xml

```diff
@@ -93,10 +93,10 @@
                 android:layout_marginStart="@dimen/marginStandardSmall"
                 android:src="@drawable/ic_certified"
                 android:visibility="gone"
-                app:layout_constraintBottom_toBottomOf="@+id/userName"
+                app:layout_constraintBottom_toBottomOf="@id/userName"
                 app:layout_constraintEnd_toEndOf="parent"
-                app:layout_constraintStart_toEndOf="@+id/userName"
-                app:layout_constraintTop_toTopOf="@+id/userName"
+                app:layout_constraintStart_toEndOf="@id/userName"
+                app:layout_constraintTop_toTopOf="@id/userName"
```

Comment:
If you want to remove a lot of badly formatted code like this, do it directly on master in another PR when your PR is already 2k+ lines